src/graphx/graphx.asm

```diff
@@ -544,8 +548,7 @@ gfx_SetPixel:
 	inc	hl
 	inc	hl
 	inc	hl			; move to next argument
-	ld	de,0
-	ld	e,(hl)			; e = y coordinate
+	ld	de,(hl)			; e = y coordinate
```

Comment:
_PixelPtr checks the full 24-bit y value in DE, but the y value is only passed as an 8-bit value to gfx_SetPixel. The upper 16 bits of DE will be undefined and possibly nonzero here, in which case I believe _PixelPtr would erroneously reject it as offscreen even if the 8-bit value is actually on screen.

Reply:
I see. Please reject it. If I remember well, I suggested that twice.